### PR TITLE
Main1.md and leadership names in place

### PR DESCRIPTION
**Pull request by alex-m-zhang**
Other changes to frontpage to come soon.

### CODE REVIEW

**Comment by rzig** [comment]
Is it possible to make the text more concise? People will glaze over that much I think - cc @gritvik

**Comment by alex-m-zhang** [comment]
I've made the mission text more concise as well.

**Comment by rzig** [comment]
Wording thoughts:
- Intro wording is funky. Feels too abrupt first paragraph and sentence about nationals is wordy (and unnecessary?)
- Minor issues in corporate with some funky wording
- Outreach should say to contact us if interested
- The social stuff feels like it should be under corporate heading

Suggested changes are below:
### Our Mission
Eta Kappa Nu (HKN) strives to assist its members—and those in the broader Illinois ECE community—throughout their lives in becoming better professionals, and encourages members to improve the standards of their profession, courses of instruction, and the institutions where they belong. In this spirit, we provide an array of services to Illinois students, including free one-on-one peer tutoring and exam review sessions for core ECE courses. HKN membership is selective, and is limited to the top (alex please insert percentages here). Prospective members, known as initiates, must earn required service and social points to join HKN.

### Corporate
HKN hosts a variety of company-sponsored events such as information sessions, tech talks, and resume reviews. Conducted on campus, these sessions give students a chance to learn more about employers, recent technological advances, and career opportunities. Company representatives are also welcome to join our pre-planned events, including our weekly happy hour, to interact with students in a more relaxed setting. Interested in working with us? Simply click the contact us button above.

### Outreach
HKN hosts outreach events, which are focused on teaching fundamental Electrical and Computer Engineering concepts to the surrounding community through application. Past examples include teaching Boy Scouts circuitry and soldering and hosting info sessions with high school students in the Champaign Public Library to build foundations in electronics. We always appreciate the opportunity to give back, and if your organization is interested in working together please reach out!

### Social
(this should be deleted!)

**Comment by rzig** [comment]
# Review Sessions Page
## Description
HKN offers many student services, the most popular of which are review sessions for nearly every core undergraduate ECE course. Review sessions are led by HKN members who are passionate about the topics they cover. Schedule, slides, recordings, worksheets, and solutions can be found below.

File: src/content/Home/hero_text.md [hunk 1]
```diff
@@ -1 +1 @@
-One sentence description of how awesome HKN is in flowery language in three lines because aesthetics.
+Promoting excellence in the Illinois ECE community with the ideals of scholarship, character and attitude.
```
Comment (lnguy7) [inline src/content/Home/hero_text.md:1]: Remove "the ideals of"

File: src/content/Home/leadership_text.md [hunk 2]
```diff
@@ -1 +1 @@
-Some text about HKN being entirely student run. Click on a leader to send them an email.
+HKN is led entirely by dedicated undergraduate and graduate students. Leaders are selected by election every semester.
```
Comment (lnguy7) [inline src/content/Home/leadership_text.md:1]: Add sth abt how we are volunteering to do this work too

**Review by lnguy7 — CHANGES_REQUESTED** [review]

File: src/content/Home/hero_text.md (hunk 1 above)
Comment (rzig) [reply src/content/Home/hero_text.md:1]: +1, also if that's removed thoughts on changing "with" to "through"?

Comment (alex-m-zhang) [reply src/content/Home/hero_text.md:1]: Seems to be resolved - this text is no longer in our mission.

File: src/content/Home/leadership_text.md (hunk 2 above)
Comment (alex-m-zhang) [reply src/content/Home/leadership_text.md:1]: Why? Isn't that sorta implied by they need to get elected?

Comment (lnguy7) [reply src/content/Home/leadership_text.md:1]: Yeah true

File: src/content/Home/main1.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 Eta Kappa Nu (HKN) strives to assist its members and those in the broader Illinois ECE community throughout their lives in becoming better professionals by encouraging its members to improve the standards of their profession, courses of instruction, and the institutions where they belong. In this spirit, we provide an array of services to Illinois students, including free one-on-one peer tutoring and exam review sessions for core ECE courses.
 
-HKN membership is selective. Prospective members, known as initiates, must not only be in the top third of their class by GPA, but must also complete 15 hours of service in a semester to demonstrate their character and attitude towards the community.
+HKN membership is selective. Prospective members, known as initiates, must not only be in the top third of their class by GPA, but must also complete at least 10 hours of service in a semester to demonstrate their character and attitude towards the community.
```
Comment (lnguy7) [inline src/content/Home/main1.md:10]: Still technically a lie but we can change that so it isn't >:)

**Review by lnguy7 — APPROVED** [review]

File: src/content/Home/main1.md [hunk 4]
```diff
@@ -5,14 +5,13 @@
 
 # Our Mission
 
-This is some text about how great HKN. I am building the website for HKN. This text should be relatively long, but not too long. I still need to refactor the code for this website and add some more new features. This is some more text, HKN does fun events like happy hour. Even more text, and even more text, and even more text! This is a long paragraph, but is necessary to make things look balanced, it's height should be not much less than that of the image below.
+Eta Kappa Nu (HKN) strives to assist its members and those in the broader Illinois ECE community throughout their lives in becoming better professionals by encouraging its members to improve the standards of their profession, courses of instruction, and the institutions where they belong. In this spirit, we provide an array of services to Illinois students, including free one-on-one peer tutoring and exam review sessions for core ECE courses.
+
+HKN membership is selective. Prospective members, known as initiates, must not only be in the top third of their class by GPA, but must also complete at least 10 hours of service in a semester to demonstrate their character and attitude towards the community.
```
Comment (rzig) [inline src/content/Home/main1.md:10]: "must not only be in the top third of their class by GPA, but must also complete" -> "must be in the top third of their class by GPA, and complete at least 10 hours of service"

Maybe write 10 as ten?

File: src/content/Home/main2.md [hunk 5]
```diff
@@ -5,15 +5,12 @@
...
-![asdf](https://www.smithgroup.com/sites/default/files/styles/slideshow_mobile_1x/public/2018-07/UIUC-ECE-10_1.jpg?h=33c22240&itok=B05fCdr5)
+![Group of students at a corporate event](/corporatepic.jpg)
 
 # Outreach
 
-HKN regularly hosts events for both members and non-members, ranging from
-tech talks to fun social gatherings like our weekly happy hour. To
-encourange involvement, HKN initiates earn points by attending events.
-Learn more about each event by clicking or tapping it.
+HKN hosts outreach events, which are focused on teaching fundamental Electrical and Computer Engineering concepts to the surrounding community through application. Past examples include holding soldering workshops and teaching Boy Scouts circuitry and hosting info sessions about electronics with high school students in the Champaign Public Library. We appreciate any opportunity to give back to the community.
```
Comment (rzig) [inline src/content/Home/main2.md:14]: "include holding soldering workshops and teaching ... and hosting" -> "include teaching Boy Scouts circuitry, holding soldering workshops, and hosting ..."

File: src/content/Home/main2.md [hunk 6]
```diff
@@ -5,15 +5,12 @@
...
 
-![asdf](https://www.smithgroup.com/sites/default/files/styles/slideshow_mobile_1x/public/2018-07/UIUC-ECE-10_1.jpg?h=33c22240&itok=B05fCdr5)
+![Group of students at a church about to give back to the community](/outreachpic.jpg)
```
Comment (rzig) [inline src/content/Home/main2.md:16]: ![Group of students at a service event]

File: src/content/StudentServices/tutoring.md [hunk 7]
```diff
@@ -1 +1 @@
-Need more personalized help? HKN offers free one-on-one peer tutoring. Simply enter your class(es) below, and contact a tutor to arrange a time.
+Need more personalized help? HKN offers free one-on-one peer tutoring. Simply enter your class(es) below, and contact a tutor by email to arrange a time. If you would like a tutor for a course that is not listed below, please reach out to our Director of Student Services and we will be happy to arrange something.
```
Comment (rzig) [inline src/content/StudentServices/tutoring.md:1]: "please reach out to our diretor of student services" -> "please contact us using the link on our homepage"

**Review by rzig — CHANGES_REQUESTED** [review]

File: src/content/Home/main2.md (hunk 6 above)
Comment (lnguy7) [reply src/content/Home/main2.md:16]: Is this value seen on the website lol

Comment (lnguy7) [reply src/content/Home/main2.md:16]: Bc please change it if it is

**Review by rzig — APPROVED** [review]
LGTM, we need to compress profile pictures though. I can do that

**Pull request merged**